modules/aggregation-layers/src/contour-layer/contour-layer.js

```diff
@@ -233,30 +219,28 @@ export default class ContourLayer extends GridAggregationLayer {
     return thresholds.some((_, i) => !equals(thresholds[i], oldThresholds[i]));
```

Comment:
Avoid defining inline functions. For loops are more efficient.

Reply:
addressed in https://github.com/uber/deck.gl/pull/3954